### ISSUE TITLE
[ros2] support VLS 128

### ISSUE THREAD

**Issue by mantkiew** [ros-drivers/velodyne#491]
Currently only VLP16 and 32C launch files are provided.

Please provide launch files from VLS-128.

Are there any technical issues preventing support for VLS-128?

**Comment by JWhitleyWork** [comment]
Yes, a fair few changes are required to support the VLS-128 - not just launch files. If you want to help, you can do so by testing https://github.com/ros-drivers/velodyne/pull/485.
